LameDLLWrap/LibMp3Lame.cs

```diff
@@ -577,24 +577,55 @@ public void ID3SetPad(int nBytes)
 		/// <param name="title">Value to set</param>
 		public void ID3SetTitle(string title)
 		{
-			NativeMethods.id3tag_set_title(context, title);
+			// XXX: Should we use Encoding.ASCII instead of UTF8?
+			// Because ID3 spec doesn't even say they support UTF8 OOTB
+			// but Latin-1 (default) and UCS2 (ID3V2 only)
```

Comment:
Encoding.ASCII replaces all non-ASCII characters with a '?' character, so it's always going to return the same byte count. UTF8 at least kind of works, in that it catches all of your CJK characters, but gives incorrect results for valid Latin1 code points with integer value >= 128.

A proper Latin1 detection would need a round-trip convert and compare like this:

```
static bool IsLatin1String(string s) => s == Encoding.Latin1.GetString(Encoding.Latin1.GetBytes(s));
```

(Not necessarily the most efficient method, but it's accurate.)